ramls/acquisitions-unit.raml

```diff
@@ -0,0 +1,48 @@
+#%RAML 1.0
+title: "Orders Storage"
+baseUri: http://github.com/folio-org/mod-orders-storage
+version: v1
+
+documentation:
+  - title: Acquisitions units
+    content: <b>CRUD APIs used to manage acquisitions units.</b>
+
+types:
+    acquisitions-unit: !include acq-models/mod-orders-storage/schemas/acquisitions_unit.json
+    acquisitions-unit-collection: !include acq-models/mod-orders-storage/schemas/acquisitions_unit_collection.json
+    UUID:
+     type: string
+     pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$
+
+traits:
```

Comment:
I believe the `validation.raml` trait should be used here as well because the implementation uses `@Validate` annotation and RMB can return 422 response if the content is incorrect e.g. required property is missed in the `POST` or `PUT` body